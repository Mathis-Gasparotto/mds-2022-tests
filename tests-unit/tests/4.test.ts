import { expect, test } from 'vitest'
import { divideNumbers } from '../4'

test('test', async () => {
  const actual = await divideNumbers(8, 2)

  expect(actual).toMatchSnapshot()
})

test('error', () => {
  // ---- La mÃ©thode ci-dessous marche aussi ----
  // try {
  //   divideNumbers(8, 0)
  // } catch (err) {
  //   const exepted = new Error('Division by zero')
  //   expect(err).toMatchSnapshot()
  // }

  expect(() => {
    divideNumbers(8, 0)
  }).toThrowError('Division by zero')
})
