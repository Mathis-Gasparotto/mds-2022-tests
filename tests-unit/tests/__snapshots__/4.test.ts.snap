// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test 1`] = `4`;
